packages/react-components/review/beta/react-components.api.md

```diff
@@ -489,17 +488,6 @@ export interface ChatMessage extends MessageCommon {
     status?: MessageStatus;
 }
 
-// @beta
-export interface ChatTheme {
```

Comment:
Did we introduce this theme just for chat overlay?

Reply:
> Did we introduce this theme just for chat overlay?

Yes, it was introduced just for chat overlay.